server/routes/chat_routes.ts

```diff
@@ -350,11 +373,16 @@ export function registerChatRoutes(router: IRouter, routeOptions: RoutesOptions)
        * Retrieve latest interactions from memory
        */
       try {
-        const conversation = await storageService.getSession(sessionId);
+        const interaction = await storageService.getInteraction(
+          sessionId,
+          outputs?.interactionId || ''
+        );
+        const messages = await storageService.getMessagesFromInteractions([interaction]);
 
         return response.ok({
           body: {
-            ...conversation,
```

Comment:
Seems some fields like `createdTimeMs` and `updatedTimeMs` was be removed in this response. Do we need to add back?

Reply:
Actually there is no consumer for `createdTimeMs ` and `updatedTimeMs`, so it is safe to remove these fields from the response of regenerate.